src/backend/functions/work-packages-create.ts

```diff
@@ -38,16 +38,20 @@ export const createWorkPackage: Handler = async ({ body }, _context) => {
       goals: true,
       features: true,
       otherConstraints: true,
+      wbsElement: true,
       workPackages: { include: { wbsElement: true, dependencies: true } }
     }
   });
 
   if (project === null) throw new TypeError('Project Id not found!');
 
   // eslint-disable-next-line prefer-destructuring
-  const { carNumber, projectNumber } = project.workPackages[0].wbsElement;
+  const { carNumber, projectNumber } = project.wbsElement;
 
-  const workPackageNumber = (await getHighestWorkPackageNumber(projectNumber)) + 1;
+  const workPackageNumber =
+    project.workPackages === undefined || project.workPackages.length === 0
```

Comment:
If you check the types, you'll see that `project.workPackages` has a type of something. like `Work_Packages & {...}` based on the prisma query, indicating that it cannot be undefined. if it ever is undefined we have prisma to blame for lying to us, so there's no need to check.

Reply:
gotcha, makes sense